crates/iota-types/src/iota_system_state/iota_system_state_inner_v1.rs

```diff
@@ -160,9 +160,9 @@ impl ValidatorMetadataV1 {
 
         let primary_address = Multiaddr::try_from(self.primary_address.clone())
             .map_err(|_| E_METADATA_INVALID_PRIMARY_ADDR)?;
-        primary_address
-            .to_anemo_address()
-            .map_err(|_| E_METADATA_INVALID_PRIMARY_ADDR)?;
+        if !primary_address.is_loosely_valid_tcp_addr() {
```

Comment:
I guess our change will break the network because I think we force the validators to already have TCP metadata?

Reply:
And even the validators can't update in the current version, because it wouldn't be a valid anemo address without UDP 😆